collect_variables/scripts/github_api/github.py

```diff
@@ -354,3 +405,36 @@ def get_data_from_api(github_url, github_owner, github_repo_name, variable_type,
 
         export_file(topics_variables, ["html_url_repository", "topic"], "topic",
                     args.topics_output)
+
+    if args.readmes:
+        # get data
+        readmes_variables = []
+        for counter, (url, owner, repo_name) in enumerate(zip(df_repos["html_url"],
+                                                              df_repos["owner"], df_repos["name"])):
+            retrieved_data = get_data_from_api(url, owner, repo_name, "readmes")
+            print(retrieved_data)
+            if retrieved_data is not None:
+                readmes_variables.append(retrieved_data)
+            if counter % 10 == 0:
+                print(f"Parsed {counter} out of {len(df_repos.index)} repos.")
+            time.sleep(SLEEP)
+
+        export_file(readmes_variables, ["html_url_repository", "readme"], "readme",
+                    args.readmes_output)
+
+    if args.coc:
+        # get data
+        coc_variables = []
+        for counter, (url, owner, repo_name,
+                      branch) in enumerate(zip(df_repos["html_url"], df_repos["owner"],
+                                               df_repos["name"], df_repos["default_branch"])):
```

Comment:
If you have an idea how this could be done in a better way I'd be happy to change it. Right now, this for loop spans over 3 lines due to linting complaining about the line length